src/solvers/dgsem_unstructured/sort_boundary_conditions.jl

```diff
@@ -8,13 +8,10 @@
 """
     UnstructuredSortedBoundaryTypes
 
-General container to sort the boundary conditions by type for the unstructured quadrilateral solver.
+General container to sort the boundary conditions by type for the unstructured meshes/solvers.
```

Comment:
I don't use this type (I just use named tuples). Should we keep "quadrilateral" in the description?

Reply:
We also use in 3D, so I don't really like including "quad" here...

Reply:
I reformulated it as "for *some* unstructured meshes/solvers". Does that sound okay?

Reply:
That sounds great